rust_snuba/rust_arroyo/src/backends/kafka/mod.rs

```diff
@@ -67,7 +67,7 @@ fn create_kafka_message(msg: BorrowedMessage) -> ArroyoMessage<KafkaPayload> {
             headers: msg.headers().map(BorrowedHeaders::detach),
             payload: msg.payload().map(|p| p.to_vec()),
         },
-        DateTime::from_utc(NaiveDateTime::from_timestamp(time_millis, 0), Utc),
```

Comment:
bugfix in arroyo: `NaiveDateTime::from_timestamp`'s first argument is seconds, not milliseconds. This caused the conversion to Python datetime to fail because the year was too large.